From f14ea9f7ff4f36fb70a85a78307686f909218159 Mon Sep 17 00:00:00 2001
From: jacob-delgado <jacob.delgado@volunteers.acasi.info>
Date: Tue, 1 Feb 2022 15:19:02 -0700
Subject: Run bin/update_deps.sh (#37099)

* Run bin/update_deps.sh

* istio/proxy cannot be merged; remove those dependencies
---
 docker/Dockerfile.distroless | 2 +-
 prow/release-commit.sh       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/docker/Dockerfile.distroless b/docker/Dockerfile.distroless
index 5325db37c3..61c5dbe5d4 100644
--- a/docker/Dockerfile.distroless
+++ b/docker/Dockerfile.distroless
@@ -1,5 +1,5 @@
 # prepare a distroless source context to copy files from
-FROM gcr.io/distroless/static-debian11@sha256:1cc74da80bbf80d89c94e0c7fe22830aa617f47643f2db73f66c8bd5bf510b25 as distroless_source
+FROM gcr.io/distroless/static-debian11@sha256:8ad6f3ec70dad966479b9fb48da991138c72ba969859098ec689d1450c2e6c97 as distroless_source
 
 # prepare a base dev to modify file contents
 FROM ubuntu:focal as ubuntu_source
diff --git a/prow/release-commit.sh b/prow/release-commit.sh
index 2c90f6f9e8..fc81f8d836 100755
--- a/prow/release-commit.sh
+++ b/prow/release-commit.sh
@@ -32,7 +32,7 @@ DOCKER_HUB=${DOCKER_HUB:-gcr.io/istio-testing}
 GCS_BUCKET=${GCS_BUCKET:-istio-build/dev}
 
 # Use a pinned version in case breaking changes are needed
-BUILDER_SHA=5c13ba8b2b7c5507cd3bd6fa60856dc5c0381694
+BUILDER_SHA=48a3d9a351c40a7afd4da2767c305861f50f5f43
 
 # Reference to the next minor version of Istio
 # This will create a version like 1.4-alpha.sha
-- 
2.35.3

